templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -509,7 +509,7 @@ Parameters:
   KeyPairName:
     ConstraintDescription: Must be the name of an existing EC2 Key Pair.
     Description: The EC2 Key Pair to allow SSH access to the instances. If you don't provide one, the Atlassian Standard Infrastructure stack's key pair will be used.
-    Type: String
+    Type: List<AWS::EC2::Keypair>
```

Comment:
This means that the `KeyPairName` is not optional anymore. You can see how it is used:
https://github.com/atlassian/quickstart-atlassian-bitbucket/blob/develop/templates/quickstart-bitbucket-dc.template.yaml#L875

```
KeyProvided:
    !Not [!Equals [!Ref KeyPairName, '']]
```
and then 
```
KeyName: !If
        - KeyProvided
        - !Ref KeyPairName
        - Fn::ImportValue: !Sub "${ExportPrefix}DefaultKey"
```
This means that having the empty string there allows users to skip filling the form with the keypair and let the cloud formation pick up the keypair from exported values.

We can make it more explicit and require the key here but then we need to change the description. I think we can keep the empty string in this case. The template with VPC can have the dropdown shown.